webpack/assets/javascripts/react_app/components/charts/ChartModal.js

```diff
@@ -0,0 +1,44 @@
+import React, {PropTypes} from 'react';
+import { Modal } from 'react-bootstrap';
+import c3 from 'c3';
+
+// destroy chart on close
+const ChartModal = ({ show, config, onHide, setTitle, title, id }) => {
+  function onEnter() {
```

Comment:
question: can't we just use the chart component with a different ID inside a model component? why do we need the duplication?